pom.xml

```diff
@@ -11,7 +11,7 @@
     <groupId>de.adorsys.keycloak</groupId>
     <artifactId>keycloak-config-cli</artifactId>
     <packaging>jar</packaging>
-    <version>5.1.1-SNAPSHOT</version>
+    <version>5.2.0-SNAPSHOT</version>
```

Comment:
Please revert this. In the current state, maven-release-plugin takes care of this, including creating a tag which triggers the action to release.

Reply:
Done